php-interop/README.md

```diff
@@ -0,0 +1,30 @@
+# PHP code for RabbitMQ tutorials based on AMQP interop
+
+Here you can find PHP code examples from [RabbitMQ
+tutorials](http://www.rabbitmq.com/getstarted.html) adopted to [amqp interop](https://github.com/queue-interop/queue-interop#amqp-interop)
```

Comment:
Please change this to say `queue-interop` since that's the name of your project and mention that it's a PHP library. `amqp-interop` can hint at AMQP 0-9-1 / AMQP 1.0 interoperability to some.